### ISSUE TITLE
Arrays always assumed to contain a $ref

### ISSUE THREAD

**Issue by languitar** [koxudaxi/fastapi-code-generator#26]
The following spec currently results in a generation error:

```yaml
openapi: 3.0.0
info:
  title: pet-finder
  version: '1.0'
servers:
  - url: 'http://localhost:3000'
paths:
  /find/:
    get:
      summary: find
      tags: []
      responses:
        '200':
          description: OK
          content:
            application/json:
              schema:
                type: array
                items:
                  type: integer
              examples:
                example-1:
                  value:
                    - 0
                    - 1
                    - 3
      operationId: get-find
components:
  schemas: {}
```

Error message:
```
❯ poetry run fastapi-codegen -i ../petfinder/reference/pet-finder.v1.yaml -o /tmp/petfinder
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/languitar/src/fastapi-code-generator/.venv/lib/python3.8/site-packages/typer/main.py", line 213, in __call__
    return get_command(self)()
  File "/home/languitar/src/fastapi-code-generator/.venv/lib/python3.8/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/home/languitar/src/fastapi-code-generator/.venv/lib/python3.8/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/home/languitar/src/fastapi-code-generator/.venv/lib/python3.8/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/languitar/src/fastapi-code-generator/.venv/lib/python3.8/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/home/languitar/src/fastapi-code-generator/.venv/lib/python3.8/site-packages/typer/main.py", line 496, in wrapper
    return callback(**use_params)  # type: ignore
  File "/home/languitar/src/fastapi-code-generator/fastapi_code_generator/__main__.py", line 26, in main
    return generate_code(input_name, input_text, output_dir, template_dir)
  File "/home/languitar/src/fastapi-code-generator/fastapi_code_generator/__main__.py", line 37, in generate_code
    parsed_object: ParsedObject = parser.parse()
  File "/home/languitar/src/fastapi-code-generator/fastapi_code_generator/parser.py", line 391, in parse
    return self.parse_paths(openapi)
  File "/home/languitar/src/fastapi-code-generator/fastapi_code_generator/parser.py", line 400, in parse_paths
    return ParsedObject(
  File "/home/languitar/src/fastapi-code-generator/fastapi_code_generator/parser.py", line 375, in __init__
    operation.response
  File "/usr/lib/python3.8/functools.py", line 967, in __get__
    val = self.func(instance)
  File "/home/languitar/src/fastapi-code-generator/fastapi_code_generator/parser.py", line 252, in response
    type_ = f'List[{schema.items.ref_object_name}]'
  File "/home/languitar/src/fastapi-code-generator/.venv/lib/python3.8/site-packages/datamodel_code_generator/parser/jsonschema.py", line 115, in ref_object_name
    return self.ref.rsplit('/', 1)[-1]  # type: ignore
AttributeError: 'NoneType' object has no attribute 'rsplit'
```

**Comment by koxudaxi** [comment]
Thank you for reporting the problem.
I will implement it :rocket:

**Comment by koxudaxi** [comment]
@languitar 
I have released a fixed version as `0.0.14`
